ros2_control_demo_hardware/src/rrbot_system_position_only.cpp

```diff
@@ -0,0 +1,175 @@
+// Copyright 2020 ROS2-Control Development Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <chrono>
+#include <cmath>
+#include <limits>
+#include <memory>
+#include <vector>
+
+#include "ros2_control_demo_hardware/rrbot_system_position_only.hpp"
+
+#include "hardware_interface/components/component_info.hpp"
+#include "hardware_interface/hardware_info.hpp"
+#include "rclcpp/rclcpp.hpp"
+
+namespace ros2_control_demo_hardware
+{
+
+return_type RRBotSystemPositionOnlyHardware::configure(const HardwareInfo & system_info)
+{
+  if (hardware_interface::BaseSystemHardwareInterface::configure(system_info) != return_type::OK) {
+    return return_type::ERROR;
+  }
+
+  hw_read_time_ = stod(info_.hardware_parameters["example_param_read_for_sec"]);
+  hw_write_time_ = stod(info_.hardware_parameters["example_param_write_for_sec"]);
+  hw_values_.resize(info_.joints.size());
+  for (uint i = 0; i < hw_values_.size(); i++) {
+    hw_values_[i] = std::numeric_limits<double>::quiet_NaN();
+  }
+  for (auto joint : info_.joints) {
```

Comment:
I'd prefer if you didn't use `auto`. Since this is an example, we should make it easy to figure out the type of these variables

Reply:
good point!